openmdao/docs/tutorials/first_analysis.rst

```diff
@@ -32,7 +32,7 @@ Preamble
 ::
 
     from __future__ import division, print_function
-    from openmdao.core.explicitcomponent import ExplicitComponent
+    from openmdao.api import ExplicitComponent
 
 At the top of any script you'll see these lines (or lines very similar to these) which import needed classes and functions. On the first import line the `print_function` is used so the code in the script will work in python 2.0 or 3.0. If you want to know whats going on with the division operator, check out this `detailed explanation <https://www.python.org/dev/peps/pep-0238/>`_. The second import line brings in OpenMDAO classes that are needed to build and run a model.
```

Comment:
I know this isn't part of your PR, but we should change python 2.0 and python 3.0 to either python 2 and python 3 or python 2.x and python 3.x.